static/src/js/components/SpatialDisplay/SpatialDisplay.js

```diff
@@ -345,6 +360,26 @@ class SpatialDisplay extends Component {
     e.preventDefault()
   }
 
+  /**
+   * Validates a Lat/Lng is limited to 5 decimal points
+   * @param {*} latLng
+   */
+  isValidDecimalLatLng(latLng) {
+    const regex = new RegExp(`^-?\\d*\\.?\\d{0,${defaultSpatialDecimalSize}},?-?\\d*\\.?\\d{0,${defaultSpatialDecimalSize}}$`)
+
+    return latLng.match(regex)
+  }
+
+  /**
+   * Validates a radius is limited to an integer
+   * @param {*} value
+   */
+  isValidRadius(value) {
```

Comment:
Methods should be prefixed with is when they return a boolean, this returns an array. I'd change the name of the method or ensure this method returns a boolean return `value.match(regex).length > 0` or something similar.